spi/w5500_spi/w5500.c

```diff
@@ -0,0 +1,268 @@
+//*****************************************************************************
+//
+//! \file w5500.c
+//! \brief W5500 HAL Interface.
+//! \version 1.0.2
+//! \date 2013/10/21
+//! \par  Revision history
+//!       <2015/02/05> Notice
```

Comment:
I'm tempted to say that we should not include this code in the example, instead provide instructions to pull it from its own repo into a preconfigred place. Including it in this fashion seems unwise.